rkt/config/auth.go

```diff
@@ -59,7 +75,7 @@ type basicAuthHeaderer struct {
 	auth basicV1
 }
 
-func (h *basicAuthHeaderer) Header() http.Header {
+func (h *basicAuthHeaderer) GetHeader() http.Header {
```

Comment:
I'm not against `Get`, you have a point, but this method has no failure condition (doesn't return errors), so it's essentially a getter, but that's a matter of taste, it's ok here :+1: